HW Assignment #1/Portfolio.html

```diff
@@ -0,0 +1,27 @@
+<center>
+	<h1>
+		<a href="About_Me.html"> ABOUT ME</a> | PORTFOLIO
+	</h1>
```

Comment:
Make sure you link both pages together using a tags. Check the solution code to see how it's done.